pgmpy/models/DynamicBayesianNetwork.py

```diff
@@ -403,6 +403,37 @@ def get_cpds(self, node=None, time_slice=0):
         else:
             return [cpd for cpd in self.cpds if set(list(cpd.variables)).issubset(self.get_slice_nodes(time_slice))]
 
+    def remove_cpds(self, *cpds):
+        """
+        Removes the cpds that are provided in the argument.
+
+        Parameters
+        ----------
+        *cpds : list, set, tuple (array-like)
+            List of CPDs which are to be associated with the model. Each CPD
+            should be an instance of `TabularCPD`.
+
+        Examples
+        --------
+        >>> from pgmpy.models import DynamicBayesianNetwork as DBN
+        >>> from pgmpy.factors import TabularCPD
+        >>> dbn = DBN()
+        >>> dbn.add_edges_from([(('D',0),('G',0)),(('I',0),('G',0)),(('D',0),('D',1)),(('I',0),('I',1))])
+        >>> grade_cpd =  TabularCPD(('G',0), 3, [[0.3,0.05,0.9,0.5],
+        ...                                      [0.4,0.25,0.8,0.03],
+        ...                                      [0.3,0.7,0.02,0.2]], [('I', 0),('D', 0)],[2,2])
+        >>> dbn.add_cpds(grade_cpd)
+        >>> dbn.get_cpds()
+        [<TabularCPD representing P(('G', 0):3 | ('I', 0):2, ('D', 0):2) at 0x3348ab0>]
+        >>> dbn.remove_cpds(grade_cpd)
+        >>> dbn.get_cpds()
+        []
+        """
+        for cpd in cpds:
+            if isinstance(cpd, tuple):
```

Comment:
@yashu-seth I already did and merged this. I was working on something which was getting blocked because of this.